.travis.yml

```diff
@@ -1,13 +1,22 @@
 language: ruby
 rvm:
 - 2.6.5
-
-# Assume bundler is being used, therefore
-# the `install` step will run `bundle install` by default.
-script: ./script/cibuild
+cache: bundler
 
 env:
   global:
   - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
 
+jobs:
+  include:
+    - stage: Run Tests
+      script: ./script/run_jekyll.sh
+      name: "Run jekyll"
+    - script: ./script/validate_json.rb
+      name: "Validate JSON"
+    - script: ./script/ping_websites.rb
+      name: "Ping websites"
+    - script: ./script/check_files_formatting.sh
+      name: "Check Files Formatting"
+
 sudo: false
```

Comment:
Since we're updating our CI config, let's also address some warnings/info messages Travis is giving us.

>  Build config validation
> root: deprecated key sudo (The key `sudo` has no effect anymore.)
> root: deprecated key sudo (The key `sudo` has no effect anymore.)
> root: missing dist, using the default xenial
> root: missing os, using the default linux

This should be fixable by removing this line and and adding the following to the top of the file:

```
os: linux
dist: xenial
```

I wouldn't be opposed to using `bionic` as the dist since that will see active development until April 2023, whereas xenial will cease active development in 2021.

Reply:
I'm currently using bionic, so yes, very stable and good for development. GTK3 got worse with this release though, irrelevant to our work though.

Reply:
FYI the `sudo: false` line should be removed as well

Reply:
Yep, I knew but totally forgot, thanks for the heads-up!

Should be good to go as soon as CI is green. I went with `bionic`, in any case.